### DISCUSSION TITLE
Public roadmap for feature support?

### DISCUSSION THREAD

**Discussion by JesseMaxwell** [bigcommerce/catalyst#650] [Q&A] [answered]
Catalyst looks like a really nice tech stack. It's great to see all the functionality that currently is present. However, there's still more work to be done. For example, the Account pages are not built out yet.

Do you have a list of supported (vs. not) features that I could reference?

**Comment by katiehoesley** [comment, marked as answer]
Hey @JesseMaxwell ! Great question. 

[This post](https://github.com/bigcommerce/catalyst/discussions/586) is pinned to this repo and will be continually updated - I'll be updating it again shortly - and it will continue to be a comprehensive list of available features. I'll also add new Announcement Discussions when new things are added. 

Do you like this approach? If you have a suggestion for another way to get this info to devs, I'm totally open to feedback and ideas! :)

**Reply by JesseMaxwell** [reply]
Thanks, @katiehoesley. In making recommendations to prospects/clients a checklist of all features that will be supported would be super helpful. For example:

- [ ] Customer Account
  - [ ] Order View (in development)
- [ ] Catalog
  - [x] Add to Compare
- [ ] B2B
  - [ ] Quotes

**Comment by nowlena** [comment]
> Thanks, @katiehoesley. In making recommendations to prospects/clients a checklist of all features that will be supported would be super helpful. For example:
> 
> * [ ]  Customer Account
>   
>   * [ ]  Order View (in development)
> * [ ]  Catalog
>   
>   * [x]  Add to Compare
> * [ ]  B2B
>   
>   * [ ]  Quotes

Agreed, being able to see the current and future feature set at-a-glance would be very helpful in scoping projects with clients.

Thanks for all the hard work! :)
